tests/unit/wifi/helpers/AccessPointControllerTest.cxx

```diff
@@ -41,6 +41,17 @@ AccessPointControllerTest::GetCapabilities()
     return AccessPoint->Capabilities;
 }
 
+bool
+AccessPointControllerTest::SetProtocol([[maybe_unused]] Microsoft::Net::Wifi::Ieee80211Protocol ieeeProtocol)
+{
+    if (AccessPoint == nullptr) {
+        throw std::runtime_error("AccessPointControllerTest::SetIeeeProtocol called with null AccessPoint");
+    }
+
+    // TODO: Set Ieee80211 protocol
```

Comment:
For this, we should add a `Ieee80211Protocol Protocol` public member to `AccessPointTest`, then set it here like this:

```suggestion
    AccessPoint->Protocol = ieeeProtocol;
```